src/main/resources/openapi/courthouse.yaml

```diff
@@ -224,6 +243,21 @@ components:
               type: string
               format: date-time
               example: "2024-02-09T14:26:31.118Z"
+    ExtendedCourthousePost:
```

Comment:
I find it less confusing if this model ended with `Response`. I was wondering initially what the id was doing in the post request, but can see it being returned in the controller. There are no other Response models in this API, so it's probably fine to leave it as is.

Reply:
Whilst I agree, I'll opt to leave it as the other response types in the schema aren't endinging with `Response`. I think it's worth revisiting as a tidy up tech task once the APIs are stable, so I'll create a ticket.